package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^0.9.5",
```

Comment:
why the version downgrade here? 👀

Reply:
I have no idea at all! I remember noticing this when I was having trouble with npm. I think this would be good to look at when merging or going over my setup in person.

Reply:
I removed this change, and nothing broke. It seems to have been from a merge from one of my oldest branches, so I did not investigate.

Reply:
I removed this change, and nothing broke. It seems to have been from a merge from one of my oldest branches, so I did not investigate.